src/common/redis_module/ray_redis_module.cc

```diff
@@ -610,6 +648,37 @@ int TableAdd_RedisCommand(RedisModuleCtx *ctx,
   }
 }
 
+/// Add an entry at a key. This overwrites any existing data at the key.
+/// Publishes a notification about the update to all subscribers, if a pubsub
+/// channel is provided.
+///
+/// This is called from a client with the command:
+///
+///    RAY.TABLE_ADD <table_prefix> <pubsub_channel> <id> <data>
+///
+/// \param table_prefix The prefix string for keys in this table.
+/// \param pubsub_channel The pubsub channel name that notifications for
+///        this key should be published to. When publishing to a specific
```

Comment:
minor, but `this` should be unindented and appear directly under `\param` (same with the next line)

Reply:
Done